### PR TITLE
QM IG updates from SANER and others

### PR DESCRIPTION
**Pull request by mholck**
FHIR-42907
FHIR-42908
FHIR-35922
FHIR-37371
FHIR-42920
FHIR-43039

### CODE REVIEW

File: input/pages/composite-measures.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Broadly speaking, composite measure scoring methods fall into two categories:
...
+Architecturally, environments that are already capable of calculating measures using the measure scoring methods already described in this implementation guide can readily consume composite measure specifications that use the first approach (individual-based) but would require additional support in order to calculate component-based measures. Specifically, completely generic support for component-based calculation methods would require that an environment be able to evaluate CQL logic in the Population context. However, by restricting composite calculation support to those methods specified by this implementation guide, environments can calculate composites by operating on the results of individual-level scores. It is important to note that each scoring method should work for any subject for which the measure is developed - e.g patient or encounter.
```
Comment (brynrhodes) [inline input/pages/composite-measures.md:40]: These are examples of basis, not subject, can we change this to e.g. patient or location?

File: input/pages/index.md [hunk 2]
```diff
@@ -2,6 +2,15 @@
 
 {:toc}
 
+<div class="new-content" markdown="1">
+<div class="note-to-balloters" markdown="1">
+#### Note To Balloters
+
+1.) If you have any examples of composite measures that use a subject other than patient please provide those so we can add them to this IG
```
Comment (brynrhodes) [inline input/pages/index.md:9]:
```suggestion
1. We are seeking examples of non-patient subject composite measures for inclusion as examples in this implementation guide.
```

**Review by brynrhodes — CHANGES_REQUESTED** [review]
Looks great, just a few minor suggestions

File: input/pages/composite-measures.md (hunk 1 above)
Comment (mholck) [reply input/pages/composite-measures.md:40]: changed to location

File: input/pages/index.md (hunk 2 above)
Comment (mholck) [reply input/pages/index.md]: Updated Note

**Comment by mholck** [comment]
all suggested changes made

File: input/pages/composite-measures.md (hunk 1 above)
Comment (abdullah-git1) [reply input/pages/composite-measures.md:40]: request changes have been made

**Review by brynrhodes — APPROVED** [review]
Looks great, nice work!

**Pull request merged**